File: mobile/lib/features/expenses/models/expense.dart
class Expense {
  const Expense({
    this.expenseId,
    required this.title,
    required this.amount,
    required this.date,
    required this.category,
  });

  final int? expenseId;
  final String title;
  final double amount;
  final DateTime date;
  final String category;
}

File: backend/ExpenseTracker/ExpenseTracker.Shared/Models/Expense.cs
namespace ExpenseTracker.Shared.Models;

public sealed class Expense
{
    public int ExpenseId { get; set; }

    public string Title { get; set; } = string.Empty;

    public decimal Amount { get; set; }

    public DateOnly ExpenseDate { get; set; }

    public string CategoryCode { get; set; } = string.Empty;
}